README.md

```diff
@@ -0,0 +1,41 @@
+# Welcome to Ubuntu Insights
+
+[actions-image]: https://github.com/ubuntu/ubuntu-insights/actions/workflows/test.yaml/badge.svg?branch=main
+[actions-url]: https://github.com/ubuntu/ubuntu-insights/actions?query=branch%3Amain+event%3Apush
+
+[license-image]: https://img.shields.io/badge/License-GPL3.0-blue.svg
+
+[codecov-image]: https://codecov.io/gh/ubuntu/ubuntu-insights/branch/master/graph/badge.svg
+[codecov-url]: https://codecov.io/gh/ubuntu/ubuntu-insights
+
+[goreport-image]: https://goreportcard.com/badge/github.com/ubuntu/ubuntu-insights
+[goreport-url]: https://goreportcard.com/report/github.com/ubuntu/ubuntu-insights
+
+[![Code quality][actions-image]][actions-url]
+[![License][license-image]](LICENSE)
+[![Code coverage][codecov-image]][codecov-url]
+[![Go Report Card][goreport-image]][goreport-url]
+
+This is the code repository for **Ubuntu Insights**, a user transparent, open, platform-agnostic and cross application solution for reporting hardware information and other collected metrics.
+
+Ubuntu Insight is designed to show you exactly what is being sent, and allow you to acknowledge and control your own data. The code in this repository is designed to mainly be invoked by a controlling application, but a command line tool is also provided.
+
+This is designed to be a full replacement for Ubuntu Report. 
+
+## Troubleshooting
+
+TODO: Add details on how to debug this project, where to increase verbosity, how to find logs, how to run in debug mode.
+
+## Get involved
+
+This is an [open source](LICENSE) project and we warmly welcome community contributions, suggestions, and constructive feedback. If you're interested in contributing, please take a look at our [Contribution guidelines](CONTRIBUTING.md) first.
+
+- to report an issue, please file a bug report against our repository, using a bug template.
+- for suggestions and constructive feedback, report a feature request bug report, using the proposed template.
+
+## Get in touch
+
+We're friendly! We have a community forum at [https://discourse.ubuntu.com](https://discourse.ubuntu.com) where we discuss feature plans, development news, issues, updates and troubleshooting.
+
+For news and updates, follow the [Ubuntu twitter account](https://twitter.com/ubuntu) and on [Facebook](https://www.facebook.com/ubuntu).
```

Comment:
Does the Ubuntu facebook/twitter link make sense here?

Reply:
See comment on bug_report